README.md

```diff
@@ -1418,6 +1418,35 @@ Other Style Guides
     }
     ```
 
+    - [13.5](#variables--unary-increment-decrement) Avoid using unary increments and decrements (++, --). eslint [`no-plusplus`](http://eslint.org/docs/rules/no-plusplus)
+
+      > Why? Per the eslint documentation, unary increment and decrement statements are subject to automatic semicolon insertion and can cause silent errors with incrementing or decrementing values within an application.  It is also more expressive to mutate your values with statements like `num += 1` instead of `num ++`.  Disallowing unary increment and decrement statements also prevents you from pre-incrementing/pre-decrementing values unintentionally which can also cause unexpected behavior in your programs.
```

Comment:
sorry to be picky, but outside of typewriters there's no need for two spaces after a period :-)